invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/side_bar/upload_info.html

```diff
@@ -0,0 +1,23 @@
+{#
```

Comment:
I would not include the upload information in the right side bar (seems a bit too prominent). We can talk IRL where to put it.